docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -633,6 +634,11 @@ Specifies the sampler to be used when sampling traces. The available samplers ar
 Specifies the argument to be passed to the sampler constructor. Must be a number.
 For const this should be 0 to never sample and 1 to always sample. _**default:**_ 1
 
+## main-snippet
+
+Adds custom configuration to the main section of the nginx configuration.
+_**default:**_ ""
```

Comment:
I think you can remove that `default: ""` from all *-snippet descriptions (including the ones that are not part of this PR). It is mentioned in the table, and is kind of obvious.

Reply:
33a284ac0c96899e018af79a39692897b236dec2